Resources/Prototypes/Recipes/Reactions/medicine.yml

```diff
@@ -277,6 +277,7 @@
 
 - type: reaction
   id: MindbreakerToxin
+  minTemp: 370
```

Comment:
37C is 310K, not 370K. did you not intend to use the temp from the removed todo?

Reply:
How does dexalin prevent lexorin poisoning?  It does not remove lexorin from the bloodstream like dexalin plus does.  It also only cures 2 asphix per unit while lexorin deals 14 per unit so it won't even have much of an effect at delaying it.

Unless I'm mistaken, won't this change make a lexorin/mindbreaker mix unstoppable?